models/articles.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+function fetchArticleById(id) {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [id])
+    .then((article) => {
+      return article;
+    });
```

Comment:
You can return `article.rows` here so you don't have to add it in the controller 🙂